apps/server/src/resources/service/controllers.spec.ts

```diff
@@ -26,7 +23,10 @@ describe('Service route', () => {
   // GET
   describe('checkServicesHealthController', () => {
     it('Should retrieve an OK services status', async () => {
-      prisma.user.upsert.mockResolvedValue(getRequestor())
+      const requestor = getRandomUser()
+      setRequestor(requestor)
+
+      prisma.project.findMany.mockResolvedValue({})
```

Comment:
Y'a besoin de ce mock ? On ne récupère pas de liste des projets sur le endpoint de l'état des services non ?

Reply:
non en effet mais je l'ai ajouté car si je n'importais pas prisma je me prenais l'erreur `Error: Failed to get openid-configuration: {"message":"certificate has expired","name":"Error","stack":"Error: certificate has expired\n` dans la CI

https://mattermost.fabrique-numerique.fr/devsecops/pl/wkxihsatajf77k4ua8wpktph7r